### PR TITLE
Canvas connect

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-1310

### CODE REVIEW

File: packages/config/src/discovery/starkware/getSHARPVerifier.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const SHARP_VERIFIER_PROXY = discovery.getMainContractDetails(
 
 const SHARP_VERIFIER = discovery.getMainContractDetails(
   'SHARPVerifier',
-  'Starkware SHARP verifier used collectively by StarkNet, Sorare, Immutable X, Apex, Myria and rhino.fi. It receives STARK proofs from the Prover attesting to the integrity of the Execution Trace of these Programs including correctly computed L2 state root which is part of the Program Output.',
+  'Starkware SHARP verifier used collectively by StarkNet, Sorare, ImmutableX, Apex, Myria, rhino.fi and Canvas Connect. It receives STARK proofs from the Prover attesting to the integrity of the Execution Trace of these Programs including correctly computed L2 state root which is part of the Program Output.',
```
Comment (antooni) [inline packages/config/src/discovery/starkware/getSHARPVerifier.ts:15]: future idea: get this one automatically from discovery

File: packages/config/src/layer2s/canvasconnect.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+)
+
+export const canvasconnect: Layer2 = {
+  type: 'layer2',
+  id: ProjectId('canvasconnect'),
+  display: {
+    name: 'Canvas Connect',
+    slug: 'canvasconnect',
+    warning:
+      'Canvas Connect is initially open only to whitelisted institutional clients.',
```
Comment (antooni) [inline packages/config/src/layer2s/canvasconnect.ts:44]:
```suggestion
      'Canvas Connect is currently open only to whitelisted addresses.',
```

File: packages/config/src/layer2s/canvasconnect.ts [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    description:
+      'Canvas Connect is a Layer 2 ZK network built on the StarkEx technology that aims to offer privacy-focused solutions for the finance and digital economy. It aims to enable the transfer, trade, and investment of tokenized real-world assets, stablecoins, CBDCs, and digital assets with assurances of privacy, low cost, and high speed.',
```
Comment (antooni) [inline packages/config/src/layer2s/canvasconnect.ts:46]: Make this less marketingy, get rid of what it aims to and state what it is, Layer2 scaling solution using StarkEx technology developed by StarkWare etc...

File: packages/config/src/layer2s/canvasconnect.ts [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  config: {
+    escrows: [
+      discovery.getEscrowDetails({
+        identifier: 'StarkExchange',
+        sinceTimestamp: new UnixTime(1675209600),
+        tokens: ['ETH', 'USDC'],
+      }),
+    ],
+  },
+  riskView: makeBridgeCompatible({
```
Comment (antooni) [inline packages/config/src/layer2s/canvasconnect.ts:71]: Are you sure about the risks? maybe some references to source code?

**Review by antooni — CHANGES_REQUESTED** [review]
Good work, some small adjustments needed

File: packages/config/src/layer2s/canvasconnect.ts (hunk 4 above)
Comment (michalsidzej) [reply packages/config/src/layer2s/canvasconnect.ts:71]: I'm doing a bulk for all starkex, let me include it there

File: packages/config/src/discovery/starkware/getSHARPVerifier.ts (hunk 1 above)
Comment (michalsidzej) [reply packages/config/src/discovery/starkware/getSHARPVerifier.ts:15]: Yep, definietly

**Review by antooni — APPROVED** [review]
LGTM
the only thing missing for me is the description for StarkExchange, but we can live without that

**Pull request merged**